linera-storage-service/src/server.rs

```diff
@@ -224,11 +225,14 @@ enum ServiceStoreServerOptions {
 
 #[tonic::async_trait]
 impl StoreProcessor for ServiceStoreServer {
+    #[allow(clippy::blocks_in_conditions)]
```

Comment:
Should we add `#![allow(clippy::blocks_in_conditions)]` at the beginning of the file instead?

Reply:
Done.